Formula/perl.rb

```diff
@@ -81,12 +72,13 @@ def caveats; <<-EOS.undent
 
     You can set that up like this:
       PERL_MM_OPT="INSTALL_BASE=$HOME/perl5" cpan local::lib
-      echo 'eval "$(perl -I$HOME/perl5/lib/perl5 -Mlocal::lib)"' >> #{shell_profile}
+      echo 'eval "$(perl -I$HOME/perl5/lib/perl5 -Mlocal::lib)"' >> #{Utils::Shell.profile}
```

Comment:
If the change is coming it seems reasonable to scoop it up before it actually does become a shouty warning, but whatever makes you happy.